tests/countries/test_cook_islands.py

```
@@ -0,0 +1,339 @@
+        self.assertHolidayName(name, (f"{year}-03-13" for year in range(2000, 2012)))
+        obs_dt = (
+            "2004-03-15",
+            "2005-03-14",
+            "2010-03-15",
+            "2011-03-14",
+        )
+        self.assertHolidayName(f"{name} (observed)", obs_dt)
+        self.assertNoHolidayName(name, range(2012, 2050))
+        self.assertNoHolidayName(f"{name} (observed)", range(2012, 2050))
+
+    def test_palmerston_gospel_day(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)